pmlc/dialect/abi/transforms/passes.td

```diff
@@ -0,0 +1,20 @@
+#ifndef __PMLC_DIALECT_ABI_PASSES__
+#define __PMLC_DIALECT_ABI_PASSES__
+
+include "mlir/Pass/PassBase.td"
+
+def LowerToABIPass : Pass<"lower-to-abi", "mlir::ModuleOp"> {
```

Comment:
The convention is for passes defined in a dialect to have the command line arg be prefixed with the name of the dialect. So it would be: `--abi-lower-to-abi` in this case.

Reply:
Done